sdk/loadtestservice/azure-resourcemanager-loadtestservice/src/test/java/com/azure/resourcemanager/loadtestservice/TestOrchestrator.java

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.loadtestservice;
+
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.Region;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.core.test.TestBase;
+import com.azure.core.test.annotation.DoNotRecord;
+import com.azure.identity.DefaultAzureCredential;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.resourcemanager.resources.ResourceManager;
+
+import java.util.Random;
+import org.junit.jupiter.api.Test;
+
+public class TestOrchestrator extends TestBase {
+
+    private static final Random RANDOM = new Random();
+    private static final Region Location = Region.US_WEST2;
+    private static final String ResourceGroupName = "az-java-sdk-"+RANDOM.nextInt(1000);
+    private static final String LoadTestResourceName = "loadtest-resource"+RANDOM.nextInt(1000);
+    private static final String QuotaBucketName = "maxEngineInstancesPerTestRun";
+
+    private DefaultAzureCredential Credential;
+    private AzureProfile Profile;
+    private LoadTestManager LoadTestsManager;
+
+    public void SetupCredential(){
+        Credential = new DefaultAzureCredentialBuilder().build();
+    }
+
+    public void SetupProfile(){
+        Profile = new AzureProfile(AzureEnvironment.AZURE);
+    }
+
+    public void PrepareTests(){
+        SetupCredential();
+        SetupProfile();
+
+        LoadTestsManager = LoadTestManager
+        .configure()
+        .authenticate(Credential, Profile);
+        
+        ResourceManager
+        .authenticate(Credential, Profile)
+        .withDefaultSubscription()
+        .resourceGroups()
+        .define(ResourceGroupName)
+        .withRegion(Location.toString())
+        .create();
+    }
+
+    public void CleanupTests(){
+        ResourceManager
+        .authenticate(Credential, Profile)
+        .withDefaultSubscription()
+        .resourceGroups()
+        .beginDeleteByName(ResourceGroupName);
+    }
```

Comment:
You should be using the "AZURE_RESOURCE_GROUP_NAME" resource group, instead of create/delete new one.

Reply:
Have updated the tests to use AZURE_RESOURCE_GROUP_NAME in the environment config